rectools/dataset/features.py

```diff
@@ -450,5 +450,21 @@ def __len__(self) -> int:
         """Return number of objects."""
         return self.values.shape[0]
 
+    def get_cat_features(self) -> "SparseFeatures":
+        """Return `SparseFeatures` only with categorical features.""" ""
+        cat_features_names_list: tp.List[SparseFeatureName] = []
+        ids_cat_features_names = []
+        for idx, (name, value) in enumerate(self.names):
+            if value != DIRECT_FEATURE_VALUE:
+                feature_name: SparseFeatureName = (name, value)
+                cat_features_names_list.append(feature_name)
+                ids_cat_features_names.append(idx)
+
+        cat_features_names_tuple: tp.Tuple[SparseFeatureName, ...] = tuple(cat_features_names_list)
```

Comment:
It's not the best practice to put var type into its name, you don't really need it here. Actually you don't really need this var at all

Reply:
I agree. I removed that part of the code